_docs/developers/knownIssues.md

```diff
@@ -7,10 +7,41 @@ tags: developers
 
 There are a few known issues we have come across due to the software upstream of QRL. While we strive to overcome programmatically any issues, sometimes there are situations where limitations exist outside of the QRL scope. This document is intended to clear up any issues seen when installing and running The QRL software. 
 
-It should be made clear that these are not "Bugs" and there is very little we can do to mitigate these limitations. We strive to work through and provide secure, audited code in any of the official channels we operate.
+It should be made clear that these are not "Bugs" and there is very little we can do to mitigate these limitations. We strive to work through and provide secure, audited code in all of the official channels we operate.
```

Comment:
"These are not bugs" where the next section talks about "There are a few small bugs".  This introduction should discuss that our project sits within an evolving ecosystem and as such there are likely to be issues that arise: some within our control and some without.  Individual components of our open source work have issues tracked on Github, and those which are important enough to need a wider audience than the developer community are documented here.

Reply:
Updated to align with above.